src/Gateway/Events/MessageCreate.php

```diff
@@ -16,7 +17,7 @@
 class MessageCreate extends Message
 {
     /**
-     * @var \Ragnarok\Fenrir\Parts\User[]
+     * @var User[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)